api/app/schemas/hfi_calc.py

```diff
@@ -23,6 +23,7 @@ class StationDaily(BaseModel):
     danger_class: Optional[int] = None
     observation_valid: Optional[bool] = None
     observation_valid_comment: Optional[str] = None
+    ros: Optional[float] = None
```

Comment:
Just thinking of future developers - what about?:
```suggestion
    rate_of_spread: Optional[float] = None
```